app/src/main/res/values/strings.xml

```diff
@@ -130,4 +130,22 @@
     <string name="change_password">Change Password</string>
     <string name="password_updated">Password updated successfully!</string>
     <string name="current_new_password_should_not_same">Current and New passwords should not be same</string>
+    <string name="about">About</string>
+    <string name="about_systers">Systers is an international community for all women involved in the technical aspects of computing. We welcome the participation of women technologists of all ages and at any stage of their studies or careers.
+We engage our community by contributing to open source, collaborating with the global community, and learning/enhancing our coding skills. We are committed to providing a safe, positive online community for our many volunteers that offer their skills, time, and commitment to our projects. </string>
+    <string name="about_app">Mentorship System is an application that matches women in tech to mentor each other, on career development, through 1:1 relations during a certain period of time.</string>
+    <string name="our_github_link">Our Github Link</string>
+    <string name="github_repository">Github Repository</string>
+    <string name="our_website">Our Website</string>
+    <string name="systers_open_source_slack">Systers Open Source Slack</string>
+    <string name="systers_open_source_website">Systers Open Source Website</string>
+    <string name="terms_and_conditions">Terms and Conditions</string>
+    <string name="code_of_conduct">Code of Conduct</string>
+    <string name="privacy_policy">Privacy Policy</string>
+    <string name="url_github">https://github.com/systers/mentorship-android</string>
```

Comment:
```suggestion
    <string name="url_systers_github">https://github.com/systers</string>
```
Maybe it would be better to redirect users to the community, and then on the Systers GitHub people can see mentorship-android project